system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts

```diff
@@ -1,56 +1,60 @@
 ### Networking-DVS Alerts ###
 
-ALERT OpenstackDVSPolling
-  IF (min(rate(openstack_networking_dvs_plugins_ml2_drivers_mech_dvs_agent_dvs_agent_process_ports_timer_count[5m])) by (cluster) *60) < 1
+ALERT OpenstackNovaDVSPolling
+  IF (min(rate(openstack_networking_dvs_plugins_ml2_drivers_mech_dvs_agent_dvs_agent_process_ports_timer_count[5m])) by (region,kubernetes_pod_name) *60) < 1
```

Comment:
What I meant is that the region will always only be a single value. We have an Prometheus in each region and each is alerting for just that region. Grouping something by region will do nothing... You can just remove the region and it will have no effect. Or am I missing something?